gatling/README.md

```diff
@@ -0,0 +1,6 @@
+# Gatling
+
+More information about Gatling tool can be found here: https://gatling.io/
+
+* `make run.gatling` runs Gatling
```

Comment:
Could maybe just expand a tiny bit on what "runs Gatling" means. Something like "ONLY RUN IF YOU WANT TO PERFORM A LOAD TEST AGAINST THE CONFIGURED ENVIRONMENT. Runs Gatling load injector for performance and capacity testing of the configured environment."